python/mspasspy/db/database.py

```diff
@@ -216,7 +216,9 @@ def read_data(
         alg_id="0",
         define_as_raw=False,
         retrieve_history_record=False,
+        method=0,
```

Comment:
Sorry that I should've thought about this: we should probably name it "merge_method" instead. Probably should also add "merge_" to the next two parameters. The names are confusing as this is the read_data method.

Reply:
Yeah I agree that it's a bit confusing and change to `merge_` it makes sense.